public/404.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Single Page Apps for GitHub Pages</title>
```

Comment:
I guess that's not a good name for snek.

This name will apear in meta information when a link is posted in a messenger, facebook, linkedin, ... or any other form that fetches meta data.

The favicon and all other meta tags have to be added to this file as well.